docs/user-guide/release-notes/2.2.x.rst

```diff
@@ -8,4 +8,9 @@ See the :fixedbugs:`2.2.0`.
 
 The docker rsync distributor has been improved to perform incremental publishes in cases where
 repositories have experienced only additive changes since the previous publish with the rsync
-distributor.
\ No newline at end of file
+distributor.
+
+A new command, docker repo tag, can point tags to manifests. It accepts a
+`tag-name` and a `manifest-digest`. If a tag with the name `tag-name` does not
```

Comment:
Also please mention `--repo-id` option, so user know the tag is created/updated within the repo